applications/wg-easy/Taskfile.yaml

```diff
@@ -221,24 +261,38 @@ tasks:
         done
       
       - echo "Release files prepared in ./release/ directory"
-      
+    deps:
+      - update-version
+
+
   release-create:
     desc: Create and promote a release using the Replicated CLI
     silent: false
-    vars:
-      CHANNEL: '{{.CHANNEL | default "Unstable"}}'
-      RELEASE_NOTES: '{{.RELEASE_NOTES | default "Release created via task release-create"}}'
+    dotenv: ['.env']
     cmds:
-      - echo "Creating and promoting release for {{.APP_NAME}} to channel {{.CHANNEL}}..."
+      - echo "Creating and promoting release for $APP_NAME to channel $CHANNEL..."
       - |
         # Create and promote the release in one step
         echo "Creating release from files in ./release directory..."
-        replicated release create --app {{.APP_NAME}} --yaml-dir ./release --release-notes "{{.RELEASE_NOTES}}" --promote {{.CHANNEL}}
-        
-        echo "Release created and promoted to channel {{.CHANNEL}}"
+        replicated release create --app $APP_NAME --yaml-dir ./release --release-notes "$RELEASE_NOTES" --promote $CHANNEL --version $VERSION
+        echo "Release version $VERSION created and promoted to channel $CHANNEL"
```

Comment:
What's the benefit / advantage of moving this out of taskfile variables?

In this case I like not using the environment mainly because I'm concerned about a users environment being tainted. For example, someone forgot they exported an app name before lunch and then push this app over top of a different application b/c they didn't close their terminal.

The Taskfile variables are automatically set to the right value and _can_ be adjusted at runtime which to me fit well. However, I'm curious if you have thoughts on why you're changing this.

Reply:
Basical, moving variables back to .env. The intension is use .env to pass all the vars from a static file which can be tracked. If they are keep using the same .env file without any change, everything should constant

Reply:
Same comment as above, I'm not a fan of moving things to the ENV or moving things to a file in the repo. 

I personally recommend checking out https://direnv.net/ that's what I use, but I don't check it in. This helps me set _and remove_ my environment based on my file path. But I don't check it into the repo.

I'm undecided if Taskfile variables or ENV is better but I tend to lean toward Taskfile as a first pass purely from a "try using the tool with it's provided features before starting to develop your own path". Maybe the Taskfile design is this way intentionally and we don't need to reinvent it.